examples/gaussian_process/plot_gpr_noisy.py

```diff
@@ -39,7 +39,8 @@
 kernel = 1.0 * RBF(length_scale=100.0, length_scale_bounds=(1e-2, 1e3)) \
     + WhiteKernel(noise_level=1, noise_level_bounds=(1e-10, 1e+1))
 gp = GaussianProcessRegressor(kernel=kernel,
-                              alpha=0.0).fit(X, y)
+                              alpha=0.0, n_jobs=2,
+                              n_restarts_optimizer=3).fit(X, y)
```

Comment:
General guideline: if there is not a strong reason behind a change, just keep it as it was before. The more focused the changes, the easier and more pleasant it is to review them.